<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>990</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GLIA </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>951</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QWidget" name="widgetRoot" native="true">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>400</height>
        </size>
       </property>
       <widget class="QTextBrowser" name="tbBrowser">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>501</y>
          <width>501</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>590</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="widgetControl">
        <attribute name="title">
         <string>系统控制</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="bnStartGrab">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>开始采集</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QPushButton" name="bnShow2D">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>绘制相位图像</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="bnStartStepGrab">
           <property name="text">
            <string>开始软采集</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QCheckBox" name="cbDoubleCam">
           <property name="text">
            <string>双机位模式</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="lInfo">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:16pt;&quot;&gt;基于广义锁相检测技术的光学干涉仪&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:16pt;&quot;&gt;解调软件设计&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:16pt;&quot;&gt;陈嘉宇&lt;br/&gt;物理与光电工程学院&lt;br/&gt;指导教师：朱云龙教授&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widgetCam">
        <attribute name="title">
         <string>相机</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QWidget" name="widgetArt" native="true">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>325</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
